packages/framework-core/src/services/read-model-store.ts

```diff
@@ -94,7 +95,8 @@ export class ReadModelStore {
     projectionMetadata: ProjectionMetadata<EntityInterface>,
     readModelName: string,
     readModelID: UUID,
-    sequenceKey?: SequenceKey
+    sequenceKey?: SequenceKey,
+    entityTypeName?: string
```

Comment:
Super minor: Would it make sense to name this param as `lastProjectedEntityName` or something similar?

Reply:
Makes sense